packages/base-driver/lib/jsonwp-proxy/protocol-converter.js

```diff
@@ -183,15 +183,20 @@ class ProtocolConverter {
   async proxySetValue(url, method, body) {
     const bodyObj = util.safeJsonParse(body);
     if (_.isPlainObject(bodyObj) && (util.hasValue(bodyObj.text) || util.hasValue(bodyObj.value))) {
-      let {text, value} = bodyObj;
+      let {text, value, mask} = bodyObj;
+
+      mask = _.isBoolean(mask) && mask === true;
+
       if (util.hasValue(text) && !util.hasValue(value)) {
         value = _.isString(text) ? [...text] : _.isArray(text) ? text : [];
         this.log.debug(
-          `Added 'value' property ${JSON.stringify(value)} to 'setValue' request body`
+          `Added 'value' property ${mask ? '**MASKED**' : JSON.stringify(value)} to 'setValue' request body`,
```

Comment:
I would say it would be ok to exclude actual values from this log line completely